Tasks/AzureAppServiceSettingsV1/azurewebappsettings.ts

```diff
@@ -0,0 +1,58 @@
+import tl = require('azure-pipelines-task-lib/task');
+import path = require('path');
+import { AzureEndpoint } from 'azurermdeploycommon/azure-arm-rest/azureModels';
+import { AzureRMEndpoint } from 'azurermdeploycommon/azure-arm-rest/azure-arm-endpoint';
+import { AzureAppService } from 'azurermdeploycommon/azure-arm-rest/azure-arm-app-service';
+import { AzureAppServiceUtility } from 'azurermdeploycommon/operations/AzureAppServiceUtility';
+import { AzureResourceFilterUtils } from './operations/AzureResourceFilterUtils';
+
+async function main() {
```

Comment:
Check task name with PMs as it will be used in yaml

Reply:
Checked.